.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,10 @@
 ## Checklist
 
-* [ ] The links are in the correct format: ``[Title of the Article](link), by [@author/`s Twitter username](link-for-twitter)``
-* [ ] I added my self to the credits with my GitHub account: ``[GitHub account](link-to-github]``
+* [x] The links are in the correct format: ``[Title of the Article](link), by [@author/`s Twitter username](link-for-twitter)``
+* [x] I added my self to the credits with my GitHub account: ``[GitHub account](link-to-github]``
```

Comment:
Could you take the opportunity and fix the bracket in `(link-to-github]` to `(link-to-github)`?.. 
it drives me crazy 😅

Reply:
I will fix that... 😂